cdap-proto/src/test/java/co/cask/cdap/proto/security/CheckAuthorizedRequestTest.java

```diff
@@ -38,7 +39,7 @@ public void testValidation() {
     }
 
     try {
-      new CheckAuthorizedRequest(Ids.namespace("foo"), "bob", null);
+      new CheckAuthorizedRequest(Ids.namespace("foo"), bob, null);
       Assert.fail();
```

Comment:
I disagree. When it is easy to tell the cause of the failure by looking at test code, messages are not necessary IMHO. Messages help when there are actual and expected values that don't match, and there is no way to tell even by looking at test code what the reason for the failure was.